UnityProject/Assets/Scripts/Devices/RemoteSignaller.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Atmospherics;
+using Chemistry;
+using UnityEngine;
+using Mirror;
+using UnityEngine.Events;
+
+[RequireComponent(typeof(Pickupable))]
+[RequireComponent(typeof(RadioMessager))]
+[RequireComponent(typeof(RadioReceiver))]
+[RequireComponent(typeof(HackingDevice))]
+public class RemoteSignaller : NetworkBehaviour, IInteractable<HandActivate>, IServerSpawn
```

Comment:
Maybe add a short description of what the RemoteSignaler is?

Also... I think it's "Signaler" with a sinlgle L, not "Signa**ll**er"